helm/polaris/values.yaml

```diff
@@ -0,0 +1,366 @@
+# -- The number of replicas to deploy (horizontal scaling).
+# Beware that replicas are stateless; don't set this number > 1 when using in-memory meta store manager.
+replicaCount: 1
+
+image:
+  # -- The image repository to pull from.
+  repository: localhost:5001/polaris
+  # -- The image pull policy.
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag.
```

Comment:
nit: why `Overrides`?

Reply:
For pullPolicy (in that case "Always")? If we do set it to always, it will need to pull down about 500MB every time.

Reply:
I simply means that we set (or define) the tag here, we do not "override", right? Is there another property for the tag?

Reply:
Oh, I see what you mean. So I set the default tag set on the `Deployment` as well as `Job`. Updated the wording.